Specification/BrAPI-Schema/BrAPI-Germplasm/Cross.json

```diff
@@ -114,9 +107,9 @@
                 "primaryModel": true
             }
         },
-        "CrossAttributes": {
+        "CrossAttribute": {
             "properties": {
-                "attributeDbId": {
```

Comment:
No DbId in CrossAttribute in v2.1. Perhaps there should be @BrapiCoordinatorSelby, something to consider for v2.2